CTN-001.tex

```diff
@@ -74,14 +76,16 @@ \section{Introduction}
 Figure \ref{fig:focal_plane_3} shows the LSSTCam photographed in the LSST clean room, with the camera rotated 90 deg clockwise with respect to the diagrams in Figures \ref{fig:focal_plane_1} and \ref{fig:focal_plane_2}.
 For more comprehensive technical details on the layout of the CCDs in the focal plane, consult the reference document LCA-13381 \citep{lca13381}. 
 
+Corresponding diagrams and a photo for ComCam are shown in Figures \ref{fig:comcam_focal_plane_1}, \ref{fig:comcam_focal_plane_2}, and \ref{fig:comcam_focal_plane_3}. 
+
 The source code used to generate the figures is available in the GitHub repository associated with this Camera Technical Note.
 
 \clearpage
 
 \begin{figure}
   \centering
   \includegraphics[width=\textwidth]{figures/LSSTCam_focal_plane_CTN_001_FIG1.pdf}
-  \caption{Focal plane layout. The list of ``dead'' (readout noise $> 4\,\mathrm{e}^{-}$ or anomalous Photon Transfer Curve gain) 
+  \caption{LSSTCam focal plane layout. The list of ``dead'' (readout noise $> 4\,\mathrm{e}^{-}$ or anomalous Photon Transfer Curve gain) 
```

Comment:
The caption needs some work.  Readout noise > 4 e- does not signify "dead".  The first group you list seem to be the Dead and Sometimes Dead from Tables 8 and 9, and the second group seems to be the High Noise ones, except R44_SG0_C02 is not listed